modules/rhcos-enable-node-scan.adoc

```diff
@@ -9,7 +9,7 @@
 If you use {ocp}, you can enable scanning of Red Hat Enterprise Linux CoreOS (RHCOS) nodes for vulnerabilities by using {rh-rhacs-first}.
 
 .Prerequisites
-* You must use {ocp} 4.10 or later for scanning RHCOS node hosts.
+* You must use {ocp} 4.10 or later for scanning RHCOS node hosts. For more details, see link:https://access.redhat.com/node/5822721[{product-title} Support Policy].
```

Comment:
Having fixed `4.10` here is probably ok, but the problem is that this item isn't specific enough. It must be a **secured cluster** of OCP 4.10+ for RHCOS node scanning to work **for that cluster**.
It does not matter where Central is running and it does not matter whether there are other non-OCP secured clusters.